tests/page-objects/reports/reports.wdio.page.js

```diff
@@ -19,8 +20,12 @@ const formTitle = () => $('#report-form #form-title');
 const submitButton = () => $('#report-form .form-footer .btn.submit');
 
 const forms = () => $$('.action-container .general-actions .actions.dropup .dropdown-menu li');
+const deselectReport = () => $('#reports-content .report-body .deselect');
+const itemSummary = () => $('#reports-content .report-body .item-summary');
+const checkCss = 'input[type="checkbox"]';
 
-const sentTask = async () => (await reportBody()).$('ul .task-list .task-state .state');
+const sentTask = async () => (await reportBodyDetails()).$('ul .task-list .task-state .state');
+const reportsByUUID = (uuid) => $$(`li[data-record-id="${uuid}"]`);
```

Comment:
I don't think you can ever have multiple reports with the same uuid .... 
Then only way that could happen is if we would somehow show the same reports twice in the list, which would be a pretty serious bug. 
I think you can change this to select a single element.